samples/affinidi-nextjs-nextauthjs/src/lib/clients/iota.ts

```diff
@@ -2,27 +2,97 @@ import {
   Configuration,
   ConfigurationsApi,
   PexQueryApi,
+  IotaApi,
+  CallbackApi,
+  IotaConfigurationDtoModeEnum,
+  InitiateDataSharingRequestOKData,
+  FetchIOTAVPResponseOK,
 } from "@affinidi-tdk/iota-client";
+import { v4 as uuidv4 } from "uuid";
+import * as jose from "jose";
 import { getAuthProvider } from "./auth-provider";
 
 export async function listIotaConfigurations() {
```

Comment:
We could name this function `listIotaWebsocketConfigurations` to make it more meaningful in terms of the return value.